prow/plugins/config.go

```diff
@@ -638,14 +639,32 @@ func (r RequireMatchingLabel) Describe() string {
 // a trigger can be listed for the repo itself or for the
 // owning organization
 func (c *Configuration) TriggerFor(org, repo string) Trigger {
+	orgRepo := fmt.Sprintf("%s/%s", org, repo)
 	for _, tr := range c.Triggers {
 		for _, r := range tr.Repos {
-			if r == org || r == fmt.Sprintf("%s/%s", org, repo) {
+			if r == org || r == orgRepo {
 				return tr
 			}
 		}
 	}
-	return Trigger{}
+	tr := Trigger{}
+	tr.SetDefaults()
+	return tr
+}
+
+var warnElideSkippedContexts time.Time
+
+func (t *Trigger) SetDefaults() {
+	truth := true
+	if t.ElideSkippedContexts == nil {
+		t.ElideSkippedContexts = &truth
+	} else if !*t.ElideSkippedContexts {
```

Comment:
Why don't we want to warn if it's set to true? The deprecation will be a noop  but stuff like check config will fail due to an unknown field

Reply:
Good call. Fixed.